include/votca/csg/basebead.h

```diff
@@ -148,7 +146,7 @@ inline const TOOLS::vec &BaseBead::getPos() const {
          "Cannot get bead position as it has not been set.");
   return bead_position_;
 }
-}  // namespace csg
-}  // namespace votca
+} // namespace csg
```

Comment:
@JensWehner @JoshuaSBrown just FYI, these whitespace changes in comments are due to a different clang-format version!

Reply:
okay so what do we do now?

Reply:
Fixing yourselves to one version of clang-format!

Reply:
@JensWehner I'm using: clang-format version 6.0.0-1ubuntu2 (tags/RELEASE_600/final)
what version is yours?